pipeline/5/Jenkinsfile-tier-0.groovy

```diff
@@ -0,0 +1,68 @@
+/*
+    Pipeline script for executing Tier 0 test suites for RH Ceph 5.0.
+*/
+// Global variables section
+
+def nodeName = "centos-7"
+def cephVersion = "pacific"
+def testStages = ['deploy': {
+                    stage('Deployment suite') {
+                        script {
+                            withEnv([
+                                "sutVMConf=conf/inventory/rhel-8.3-server-x86_64.yaml",
+                                "sutConf=conf/${cephVersion}/cephadm/sanity-cephadm.yaml",
+                                "testSuite=suites/${cephVersion}/cephadm/cephadm_deploy.yaml",
+                                "addnArgs=--post-results --log-level DEBUG"
+                            ]) {
+                                def sharedLib = load("${env.WORKSPACE}/pipeline/vars/common.groovy")
+                                sharedLib.runTestSuite()
+                            }
+                        }
+                    }
+                 }, 'upgrade': {
+
+                    stage('Upgrade suite') {
+                        println "Work in progress"
+                    }
+
+                 }]
+
+// Pipeline script entry point
+
+node(nodeName) {
+
+    stage('Checkout') {
```

Comment:
Can we combine checkout and Prepare node as a single pre-requisite task.

Reply:
It's possible... I understand joining them would combine the setup process.